_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -289,6 +289,14 @@ function page_admin_scripts() {
 				'showPromotions' => apply_filters( 'jetpack_show_promotions', true ),
 				'isAtomicSite' => jetpack_is_atomic_site(),
 				'plan' => Jetpack::get_active_plan(),
+				/**
+				 * Whether UI for backups should be displayed.
+				 *
+				 * @since 6.5.0
+				 *
+				 * @param bool $show_backups Should UI for backups be displayed? True by default.
+				 */
+				'showBackups' => is_plugin_active( 'vaultpress/vaultpress.php' ) || apply_filters( 'jetpack_show_backups', true ),
```

Comment:
Use `Jetpack::is_plugin_active()` instead of `is_plugin_active` since that one gets plugins that were single or network activated.